packages/editor/src/components/widgets/collapsible/Collapsible.css

```diff
@@ -1,10 +1,10 @@
-.collapsible-root .collapsible-header {
+.collapsible-root .collapsible-header:has([data-state='open']) {
```

Comment:
`:has` is not needed here
just do `.collapse-header[data-state='open']`

And I think you should have a default style without a data-state and then the style for a specific data state e.g. 
```css
.collapsible-root .collapsible-header {
  display: flex;
}
.collapsible-root .collapsible-header[data-state='open'] {
  justify-content: space-between;
  align-items: center;
}
```

Reply:
Ah if you want to check the child state, yes then has is needed.
```css
.collapsible-root .collapsible-header {
  display: flex;
}
.collapsible-root .collapsible-header:has([data-state='open']) {
  justify-content: space-between;
  align-items: center;
}
```